docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md

```diff
@@ -26,6 +27,512 @@
 | [v1.7.1](Data_Dictionary_Release_Notes.md#release-with-api-v171) | March 16, 2017 |
 | [v1.3.1](Data_Dictionary_Release_Notes.md#release-with-api-v131) | September 7, 2016 |
 
+## v3.1.0
+
+* __GDC Product__: GDC Data Dictionary
+* __Release Date__: May XX, 2024
+
+### New Features and Changes
+
+* New Entity: `germline_mutation_index`
+* New Entity: `submitted_expression_array`
+* Altered `aggregated_somatic_mutation` Entity
+	* New property: `platform`
+* Altered `aligned_reads` Entity
+	* Added `minimum` and/or `maximum` values to properties <!-- summarized this for readability --> 
+		* `average_base_quality`
+		* `average_insert_size`
+		* `average_read_length`
+		* `contamination`
+		* `contamination_error`
+		* `mean_coverage`
+		* `msi_score`
+		* `pairs_on_diff_chr`
+		* `proportion_base_mismatch`
+		* `proportion_coverage_10x`
+		* `proportion_coverage_30x`
+		* `proportion_reads_duplicated`
+		* `proportion_reads_mapped`
+		* `proportion_targets_no_coverage`
+		* `total_reads`
+		* `tumor_ploidy`
+		* `tumor_purity`
+* Altered `annotated_somatic_mutation` Entity
+	* New property: `platform`
+* Altered `annotation` Entity
+	* Changes made to `links`
+		* `germline_mutation_indexes` added to subgroup
+		* `submitted_expression_arrays` added to subgroup
+	* Changes made to `properties`
+		* New property: `germline_mutation_indexes`
+		* New property: `submitted_expression_arrays`
+* Altered `copy_number_auxiliary_file` Entity <!-- excluded new internal prop `curated_model_index` -->
+	* Changes made to `data_format`
+		* New permissible value: `PDF`
+	* Changes made to `data_type`
+		* New permissible value: `CNV Model`
+* Altered `copy_number_segment` Entity
+	* New property: `cancer_dna_fraction`
+	* New property: `genome_doubling`
+	* New property: `subclonal_genome_fraction`
+	* New property: `tumor_ploidy`
+	* New property: `tumor_purity`
+* Altered `demographic` Entity
+	* Changes made to `country_of_birth`
```

Comment:
We might want to think about summarizing Not Reported/Unknown values in the notes, but only if we have time.

Reply:
Summarized props with Not Reported/Unknown values better